src/schema/rules/files/raw/dwi.yaml

```
@@ -43,6 +43,7 @@ sbref:
 isotropic:
```

Seems weird to call FA "isotropic". Maybe:

```suggestion
ScannerDerivatives:
```